### DISCUSSION TITLE
Рандомно не передаются данные через быструю форму

### DISCUSSION THREAD

**Discussion by dragonvedb** [totumonline/totum-ru-issues-and-discussions#621] [ОШБК!] [answered]
Добрый день!

Делаю быструю форму, где два поля скрыты и получают данные через setTotumValue. На стороне клиента всё вроде бы работает нормально и я вижу в консоли, что полям назначается верное значение

Однако после отправки формы, одна или обе ячейки в таблице иногда получают значение null. По экспериментам, примерно в 50% случаев. С чем может быть связано?

Код писал согласно документации:

let interval1 = setInterval(() => {
    let element = groupTrueInput
    if(element && element.setTotumValue){
      clearInterval(interval1)
      element.setTotumValue(JSON.stringify([selectedGroup]))
      console.log(element.value)
    }
  }, 100)

  let interval2 = setInterval(() => {
    let element = numberTrueInput
    if(element && element.setTotumValue){
      clearInterval(interval2)
      element.setTotumValue(JSON.stringify([phoneNumberInput.value]))
      console.log(element.value)
    }
  }, 100)
  
  Пробовал и вариант без интервалов, просто назначать через setTotumValue, разницы не заметил

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Возможно. Это место относящееся к разделу «слишком много функционала». Мы туда посмотрим только по запросу от пользователя с платной лицензией.
